src/cl_sii/dte/data_models.py

```diff
@@ -322,24 +311,20 @@ def deudor_rut(self) -> Rut:
     # Validators
     ###########################################################################
 
-    @pydantic.v1.validator('monto_total')
-    def validate_monto_total(cls, v: object, values: Mapping[str, object]) -> object:
-        tipo_dte = values.get('tipo_dte')
+    @pydantic.field_validator('monto_total')
+    def validate_monto_total(cls, v: object, obj: FieldValidationInfo) -> object:
```

Comment:
⚠️ This review comment applies to all similar cases.

1) > `FieldValidationInfo` is **deprecated** in 2.4, use `ValidationInfo` instead.
    Source: https://github.com/pydantic/pydantic/blob/v2.4.2/docs/concepts/validators.md?plain=1#L385-L386

2) Also, please reference `ValidationInfo` as `pydantic.ValidationInfo` so that it is easier to recognize the origin of the class.

```suggestion
    def validate_monto_total(cls, v: object, obj: pydantic.ValidationInfo) -> object:
```